model_x86.json

```diff
@@ -0,0 +1 @@
+{"model_name": "SABORR", "variables": {"ImageSize": "300,300,3", "ModelPath": "", "ClassID": "1", "MinScore": "0.25"}}
```

Comment:
IS this in root? The ml_folders should be included in the gitignore

Reply:
These files should be stored in some subfolder of the `data` folder (on that volume)

Reply:
As it is part of the config, I decided to include it in said directory: data/processor/config/sources/0/ml_models/model_x86.json.
Should I change this path? Which one would you recommend?

Reply:
That path seems good. Just this file seems to be in the root directory